distribution/packages/rpm/build.gradle

```diff
@@ -1,85 +1 @@
-/*
```

Comment:
We should probably keep the license header.

Reply:
Talking to @rjernst, I think we're fine dropping the license header but we should have a comment about how the build is configured in the project above. If we drop the file altogether and use `settings.gradle` to configure the directory then that is fine too.